resources/views/partials/navbar/navbar.blade.php

```diff
@@ -13,6 +15,16 @@
         {{-- Custom left links --}}
         @yield('content_top_nav_left')
     </ul>
+    
+    {{-- Search form top --}}
+    @foreach($adminlte->menu('navbar-left') as $item)
+        @if ($menuItemHelper->isSearchBar($item) && $item['topnav'])
+
+            {{-- Search form --}}
+            @include('adminlte::partials.navbar.menu-item-search-form')
+
+        @endif
+    @endforeach
```

Comment:
Neither need, with this you will never have the chance to put the search bar at the top left corner if you add other left items. Currently, you can manage this order from the menu configuration, and this restriction will not be nice.